settings.gradle.kts

```diff
@@ -45,6 +47,9 @@ dependencyResolutionManagement {
 
             library("database-h2", "com.h2database:h2:2.1.214")
             library("database-postgres", "org.postgresql:postgresql:42.5.0")
+
+            library("bcpkix-jdk15on", "org.bouncycastle:bcpkix-jdk15on:1.70")
+            library("bcprov-jdk15on", "org.bouncycastle:bcprov-jdk15on:1.70")
```

Comment:
Kann Bouncy Castle jetzt nicht wieder raus weil wir es nur implizit über spring security verwenden?

Reply:
Wäre auch mein Verständnis. Werde ich validieren und meine Erkenntnisse hier reinschreiben.

Reply:
Habe ich rausgenommen, ist scheinbar nichts kaputt gegangen in der CI.

Reply:
Ist doch etwas kaputt gegangen.
Philip hat es daher wieder reingenommen.

wir benutzen zwar spring security,  dieses braucht jedoch diesen Import hier.   D.h. es lädt die dependency scheinbar nicht selbst.

Reply:
alles klar